src/pages/using-spark/components/button.mdx

```diff
@@ -132,15 +96,35 @@ Represents less prominent actions. Sometimes in an action set with [Primary](#pr
   titleAttr="Tertiary Button Example"
 />
 
-## States
+### Compact Buttons
+
+A smaller version of regular sized buttons.
 
-### Disabled
+- All button types (Primary, Secondary and Tertiary) can be Compact buttons.
+- Use in smaller viewports, like mobile screens.
+- Do not mix with regular sized buttons.
```

Comment:
"regular" is an relative, and assumptive word. I would clarify this with "Primary, Secondary, or Tertiary" buttons.